docs/configure/ci/configuration.md

```diff
@@ -73,3 +73,42 @@ It's a good practice to include the `reason` field for future reference.
 * `exemptions[].report` - String - the name of the report type (`polaris`, `pluto`, `trivy` or `opa`)
 * `exemptions[].checks[]` - Array - an array of check IDs to skip (e.g. `runAsNonRoot`)
 * `exemptions[].reason` - String - a human-readable description of why this exemption is necessary
+
+## Troubleshooting and Advanced Configurations
+
+### Git Checkouts
+Insights needs to gather the following information from Git:
+* The hash of the current commit
+* The hash of the base commit
+* The commit message
+* The branch name
+* The origin URL
+
+Some CI providers only provide a partial checkout of the Git repository
+by default, and some (e.g. Google Cloud Build) check out the repository
+in "detached HEAD" state, which makes it hard to gather this information.
+
+In these cases you may see an error like
+```bash
+time="2022-08-25T16:47:49Z" level=error msg="Error running /usr/bin/git merge-base HEAD main - fatal: Not a valid object name main\n[exit status 128]"
+time="2022-08-25T16:47:49Z" level=error msg="Unable to get GIT merge-base"
+time="2022-08-25T16:47:49Z" level=fatal msg="Unable to get git details: exit status 128"
+```
+
+There are a couple ways to work around this:
+* Configure your CI provider to do a "full checkout" of the repository
+  * Jenkins
```

Comment:
Jenkins what?

Reply:
Just Jenkins 😛 

Thanks for catching this, apparently I never finished